finalPrep/README.md

```diff
@@ -18,37 +18,43 @@ Some things to consider as you refactor include...
-- [ ] I have refactored my `section2` exercises to the best of my ability
-- [ ] I have refactored my `section3` exercises to the best of my ability
-- [ ] I have refactored my `section4` exercises to the best of my ability
+- [x] I have refactored my `section1` exercises to the best of my ability
+- [x] I have refactored my `section2` exercises to the best of my ability
+- [x] I have refactored my `section3` exercises to the best of my ability
+- [x] I have refactored my `section4` exercises to the best of my ability
 
 ### Time Management Prep
 In Mod 0 you've learned about different techniques for managing your time at Turing. Please create a calendar for your **first 3 weeks of Mod 1**. Feel free to make your calendar fit your style, but we suggest that your calendar should include the following:
-- [ ] Standard M1 class schedule (see M1 calendar [here](https://frontend.turing.io/today/))
-- [ ] Study/Project work time
-- [ ] Health + Wellness
+- [x] Standard M1 class schedule (see M1 calendar [here](https://frontend.turing.io/today/))
+- [x] Study/Project work time
+- [x] Health + Wellness
 
 When you are finished, add screenshots of your calendar so we can provide feedback if needed!
 
-- `Add Week 1 Screenshot Here`
-- `Add Week 2 Screenshot Here`
-- `Add Week 3 Screenshot Here`
+- `Add Week 1 Screenshot Here` ![Week_1](https://user-images.githubusercontent.com/87670195/128588598-ba3b178f-7f77-45bf-970d-0b81cbd4915e.png)
+- `Add Week 2 Screenshot Here` ![Week_2](https://user-images.githubusercontent.com/87670195/128588597-54c383ed-2d51-4840-b193-172558f8e06e.png)
+- `Add Week 3 Screenshot Here` ![Week_3](https://user-images.githubusercontent.com/87670195/128588595-3c053974-151f-43cc-845d-4f25beab1ce0.png)
 
 ### Mentorship Prep
 Mentorship is an integral part of the Turing experience and will help jumpstart your technical career. In order to get your mentor relationship started on the right foot, please complete the following deliverables:
-- [ ] Complete the [Mentorship DTR Prep](https://gist.github.com/ericweissman/51965bdcbf42970d43d817818bfaef3c)
-  - [ ] Add link to your gist here: 
+- [x] Complete the [Mentorship DTR Prep](https://gist.github.com/ericweissman/51965bdcbf42970d43d817818bfaef3c)
+  - [x] Add link to your gist here:
+
+  [Tanya's Mentoship Prep](https://gist.github.com/tanyazhuge/964b8244d9be66132ec10429cfb73059)
 
 ### Lesson Prep
 You've learned a lot about how to take strong notes during Mod 0. Show us your skills while you learn how to pre-teach content for your first lesson in Mod 1!
-- [ ] Complete the [Pre Teaching Practice exercise](https://gist.github.com/ericweissman/0036e8fe272c02bd6d4bb14f42fd2f79) gist
-  - [ ] Add a link to your gist here:
+- [x] Complete the [Pre Teaching Practice exercise](https://gist.github.com/ericweissman/0036e8fe272c02bd6d4bb14f42fd2f79) gist
+  - [x] Add a link to your gist here:
+
+  [Tanya's Lesson Prep](https://gist.github.com/tanyazhuge/b7b0e84eb26ec9563089299d42b8ed70)
```

Comment:
Nice job. I really like how you leave plenty of extra space to add notes as needed. I think using the different colors helps make your notes more readable, but you might think about more process about using different colors- for example, do you want to use a particular color for key vocabulary definitions? Or a color for code examples?